src/Android/Xamarin.Android/ArcGISRuntime.Xamarin.Samples.Android.csproj.bak

```diff
@@ -0,0 +1,393 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I would get rid of this file. I think an update to Xamarin caused this file to start being made.